EXILED/Exiled.Events/Handlers/Internal/Round.cs

```diff
@@ -98,8 +98,13 @@ public static void OnVerified(VerifiedEventArgs ev)
             RoleAssigner.CheckLateJoin(ev.Player.ReferenceHub, ClientInstanceMode.ReadyClient);
 
             // TODO: Remove if this has been fixed for https://git.scpslgame.com/northwood-qa/scpsl-bug-reporting/-/issues/52
-            foreach (Room room in Room.List.Where(current => current.AreLightsOff))
+            foreach (Room room in Room.List)
             {
+                // Null safety check for github issue #206
+                if (room.RoomLightControllers == null)
+                    continue;
```

Comment:
It's would be better to find why a Room have a room light controller null since that would cause issue for plugin